cluster-bootstrap/src/main/scala/akka/management/cluster/bootstrap/contactpoint/HttpBootstrapJsonProtocol.scala

```diff
@@ -30,6 +30,7 @@ object HttpBootstrapJsonProtocol extends DefaultJsonProtocol {
 
   // we use Address since we want to know which protocol is being used (tcp, artery, artery-tcp etc)
   final case class ClusterMember(node: Address, nodeUid: Long, status: String, roles: Set[String])
+  implicit val clusterMemberOrdering: Ordering[ClusterMember] = Ordering.by(_.node)
```

Comment:
how did it get the ordering before?

Reply:
Before 2.12, `SortedSet.map[F, T](F => T)` would produce an unsorted `Set`.
Since 2.13, `SortedSet.map[F, T](F => T)` will produce a set that is kept sorted according the ordering on `T`.

So before 2.12 an `Ordering[ClusterMember`]` was not needed when we mapped a `SortedSet` from `Member` to `ClusterMember`. There are of course other ways to solve this at the `map` side, but those got kind of ugly or needlessly broke binary compat, so this seemed like a reasonable way to deal with it (even though we don't really care about the ordering here)

Reply:
good to know, thanks